### PR TITLE
Add button to disable longpoll (via redis)

### PR DESCRIPTION
**Pull request by jacob-pascual**

### CODE REVIEW

**Comment by briankassouf** [comment]
This might make more sense on the front end. So we don't even make connections, which would always result in a 404

**Comment by lospro7** [comment]
so this requires a restart? Not sure the specifics of this, but :+1:

File: server.go [hunk 1]
```diff
@@ -168,6 +173,13 @@ func (this *Server) ListenFromLongpoll() {
 		w.Header().Set("Connection", "keep-alive")
 		//w.Header().Set("Content-Encoding", "gzip")
 
+		longpollIsDisabled := disableLongpoll.Load()
```
Comment (briankassouf) [inline server.go:175]: Should this happen before we create the socket instance?

**Comment by briankassouf** [comment]
:+1:

File: server.go (hunk 1 above)
Comment (jacob-pascual) [inline server.go:175]: yeah, I'll just close the responsewriter directly

**Pull request merged**